core/src/test/scala/akka/persistence/r2dbc/journal/R2dbcJournalPerfManyActorsSpec.scala

```diff
@@ -17,6 +17,7 @@ import akka.persistence.journal.JournalPerfSpec.ResetCounter
 import akka.persistence.r2dbc.TestDbLifecycle
 import akka.testkit.TestProbe
 
+@org.scalatest.Ignore // TODO: Remove once data can be made json-compatible
```

Comment:
It's quite intrusive to make all test data json (or that kind of quoted strings). Can't we test this feature in some other way? Some dedicated test that is verifying that json can be stored and retrieved for the journal, snapshot and read from queries, instead of aiming for running all tests with json?

Reply:
Yes, I feel the same way. I did it this way to make sure I didn't miss any functionality where changing the payload column type might have an impact (I haven't worked on this codebase until this PR). But now that all tests pass (except the intentionally ignored), changing to specialized test (and reverting the test data changes) are probably a better idea. I'll start working on that and get back once I have something in place. Thanks for the quick feedback!